testservices/identityservice/v3userpass_test.go

```diff
@@ -0,0 +1,146 @@
+package identityservice
```

Comment:
Similarly here - to reduce the code duplication I'd suggest extracting a BaseUserPassSuite with the common code, then embed it into UserPassSuite and V3UserPassSuite.